### PR TITLE
feat: added getBlocks

### PR DESCRIPTION
**Pull request by raghukapur9**
- Added support to view all blocks on the explorer

### CODE REVIEW

File: packages/app-explorer/src/systems/Block/actions/get-blocks.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    after?: string;
+  };
+  if (cursor && dir === 'after') {
+    params.after = cursor;
+  }
+  if (cursor && dir === 'before') {
+    params.before = cursor;
+  }
+
+  const { data } = await sdk.blocks(params);
```
Comment (rodrigobranas) [inline packages/app-explorer/src/systems/Block/actions/get-blocks.ts:28]: Need to regenerate sdk using pnpm codegen, then commit the changes as we generate this sdk locally

**Comment by rodrigobranas** [comment]
also, please merge changes from main into the branch

**Comment by rodrigobranas** [comment]
Tested the action on frontend and the GraphQL endpoint and it looks OK

File: packages/app-explorer/src/systems/Block/actions/get-blocks.ts (hunk 1 above)
Comment (raghukapur9) [reply packages/app-explorer/src/systems/Block/actions/get-blocks.ts:28]: I have pushed it now

**Review by rodrigobranas — APPROVED** [review]

**Review by rodrigobranas — APPROVED** [review]

**Pull request merged**